backend/organization/tests/models/test_models_incomingorganization.py

```diff
@@ -0,0 +1,63 @@
+from typing import Type
+
+from django.contrib.auth import hashers
+from django.contrib.auth.hashers import BasePasswordHasher
+from django.contrib.auth.hashers import PBKDF2PasswordHasher
+from django.test import TestCase
+
+from organization.models import IncomingOrganization
+
+
+# Cf Django 4.1 release notes https://docs.djangoproject.com/en/dev/releases/4.1
+class PBKDF2PasswordDefault40Hasher(PBKDF2PasswordHasher):
+    algorithm = "pbkdf2_4_0"
+    iterations = 320_000
+
+
+# Cf Django 4.2 release notes https://docs.djangoproject.com/en/dev/releases/4.2
+class PBKDF2PasswordDefault41Hasher(PBKDF2PasswordHasher):
+    algorithm = "pbkdf2_4_1"
+    iterations = 600_000
+
+
+class IncomingOrganizationTests(TestCase):
```

Comment:
Add a test where both secret and password are defined .

Reply:
done!